lib/components/viewers/live-stop-times.js

```diff
@@ -0,0 +1,161 @@
+import moment from 'moment'
+import 'moment-timezone'
+import coreUtils from '@opentripplanner/core-utils'
+import React, { Component } from 'react'
+
+import Icon from '../narrative/icon'
+import PatternRow from './pattern-row'
+import {
+  getRouteIdForPattern,
+  getStopTimesByPattern,
+  routeIsValid
+} from '../../util/viewer'
+
+const { getUserTimezone } = coreUtils.time
+
+const defaultState = {
+  spin: false,
+  timer: null
+}
+
+/**
+ * Table showing next arrivals (refreshing every 10 seconds) for the specified
+ * stop organized by route pattern.
+ */
+class LiveStopTimes extends Component {
+  state = defaultState
+
+  _refreshStopTimes = () => {
+    const { findStopTimesForStop, viewedStop } = this.props
+    findStopTimesForStop({ stopId: viewedStop.stopId })
+    // TODO: GraphQL approach would just call findStop again.
+    // findStop({ stopId: viewedStop.stopId })
```

Comment:
I don't know what this means and it doesn't seem particularly relevant, so seems like a good candidate for comment removal.

Reply:
It is copy paste from exiting code. I removed the comment in https://github.com/opentripplanner/otp-react-redux/pull/292/commits/bb270a7041700a19cc42eebf190a88fc04b0214c.